### ISSUE TITLE
rbenv completion spews errors on load

### ISSUE THREAD

**Issue by faho** [fish-shell/fish-shell#6617]
From the "how-did-nobody-report-this" department:

Try to complete `rbenv`, and you get an error about there being too many arguments to `complete` in this line:

```fish
complete -f -c rbenv -n '__fish_rbenv_needs_command' -a version-file 'Detect the file that sets the current rbenv version
```

Of course the "-d" before the description is missing.

Is anybody using these completions?

Opening an issue mainly to tag it as 3.1.1 to remind us to cherry-pick it.

**Comment by faho** [comment]
Ah, wait, added in 5c8ef19ff8b0eecfa86dd79c6143984ec858227f, which was merged after 3.1.0.

**Comment by faho** [comment]
The optipng completions spew too. Sorry!
